### PR TITLE
Fix the build

### PR DESCRIPTION
**Pull request by CRogers**
## Before this PR
Build does not work yet.

## After this PR
Build works

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -27,7 +27,6 @@ allprojects {
     repositories {
         gradlePluginPortal()
```
Comment (carterkozak) [inline build.gradle:28]: Do we need to opt out of metadata here as well?
```suggestion
        gradlePluginPortal() { metadataSources { mavenPom(); ignoreGradleMetadataRedirection() } }
```

**Comment by carterkozak** [comment]
oof, the classpath conflicts are coming from baseline itself due to some eclipse workaround thing :-(

File: build.gradle (hunk 1 above)
Comment (CRogers) [reply build.gradle:28]: I think an excavator just enforces this??

Comment (CRogers) [reply build.gradle:28]: hmm, this did not do what I expected https://github.com/palantir/gradle-jdks/pull/3

**Review by carterkozak — APPROVED** [review]

**Pull request merged**